ECLIPs/ECLIP-etcbyz.md

```diff
@@ -0,0 +1,80 @@
+### ECLIP-?: Support for ETH Byzantium EVM and Protocol Upgrades
+
+    ECLIP: undecided
+    Title: Support for ETH Byzantium EVM and Protocol Upgrades
+    Status: Draft
+    Type: Standards Track
+    Author: Isaac Ardis <isaac.a@etclabs.org>
+    Created: 2019-02-11
+
+### Abstract
+
+Add support for a subset of protocol-impacting changes introduced in the Ethereum Foundation (ETH) network via the _Byzantium_ hardfork. The proposed changes include:
+
+- Byzantium EVM opcodes and precompiled contracts, namely opcodes `REVERT` (EIP 206/140), `RETURNDATASIZE` (EIP 211), `RETURNDATACOPY` (EIP 211), and `STATICCALL` (EIP 214/116); and precompiled contracts for modular exponentiation, elliptic curve addition, scalar multiplication, and pairing (EIPs 198, 212/197, 213/196)
+- Replacing the intermediate state root field in transaction receipts with the contract return status (EIP 658).
+
+This document proposes block `X,XXX,XXX` as the upcoming block height at which to implement these changes in the network, placing the expected date of protocol hardfork date on _XXXX-XX-XX_.
+
+For more information on the opcodes and their respective EIPs and implementations, please see the __History__ section of this document.
+
+### Motivation
+
+To enhance the EVM's capabilities by adding 5 opcodes and 4 precompiled contracts, all of which have been in use on the ETH network since 2017-10-16. Adoption of the "receipt status" feature provides a helpful method for Dapp developers to access the successful or failed state a contact. This would (re)establish a greater level of interoperability between Foundation and Classic Ethereum Virtual Machines ("EVM"s), and make a wider array of tooling available for the ETC network (eg. Solidity version, several contract debugging tools).
```

Comment:
```suggestion
To enhance the EVM's capabilities by adding 5 opcodes and 4 precompiled contracts, all of which have been in use on the ETH network since 2017-10-16. Adoption of the "receipt status" feature provides a helpful method for Dapp developers to access the successful or failed state of a contact. This would (re)establish a greater level of interoperability between Foundation and Classic Ethereum Virtual Machines ("EVM"s), and make a wider array of tooling available for the ETC network (eg. Solidity version, several contract debugging tools).
```

Reply:
`state a contact.` -> `state of a contact.`